core/cap-0035.md

```diff
@@ -644,6 +652,12 @@ allow an issuer to disable clawback on an asset entirely by removing `AUTH_CLAWB
 on its own account and clearing `TRUSTLINE_CLAWBACK_ENABLED_FLAG` from all existing trustlines to 
 that asset.
 
+`SetTrustLineFlagsOp` uses an `Asset` as a parameter instead of an `AssetCode`. This won't matter
+in protocol 16 because the operation is invalid if the issuer on `Asset` is not the source account,
+but it'll allow us to explore options for an account that's not the issuer to change trustline flags.
+The two scenarios we currently have in mind are the owner changing the trustline flag and the issuer
+delegating trustline flag modification to a different account.

```

Comment:
💡 Suggest rewording to focus on deferring the decision rather than catering to possible future decisions, but if you prefer the existing language I'm still 👍🏻 with it.
```suggestion
`SetTrustLineFlagsOp` uses an `Asset` as a parameter instead of an `AssetCode` to defer the decision of who all future authorizers of the operation can be. The `Asset` includes the issuer account, where-as use of `AssetCode` would assume the source account is always the issuer account. In this proposal, only the issuer may authorize the operation. This decision makes it possible in a future proposal to expand the accounts that may be allowed to issue the operation.
```